src/Symfony/Bundle/FrameworkBundle/Tests/Functional/CachePoolsTest.php

```diff
@@ -24,6 +24,7 @@ public function testCachePools()
 
     /**
      * @requires extension redis
+     * @group redis
```

Comment:
let's name the group `docker` to have only one?

Reply:
Yeah, i was also considering "integration". Will go with docker 👍

Reply:
I like "integration"!

Reply:
Integration it is! ;)